pom.xml

```diff
@@ -1622,5 +1623,62 @@
         <surefire.reuseForks>${reuseForks}</surefire.reuseForks>
       </properties>
     </profile>
+    <profile>
+      <!-- This profile uses the Google errorprone tool to perform static code analysis at
+      compile time. Auto-generated code is not checked.
+      See: https://errorprone.info/bugpatterns for list of available bug patterns.-->
+      <id>errorprone</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <showWarnings>true</showWarnings>
+              <source>${maven.compiler.source}</source>
+              <source>${maven.compiler.target}}</source>
+              <encoding>UTF-8</encoding>
```

Comment:
These are already set in the main config, so they don't need to be set here. The rest of the config here should be merged with that.
```suggestion
```